Doc/reference/compound_stmts.rst

```diff
@@ -121,7 +121,7 @@ expression is true:
 
 This repeatedly tests the expression and, if it is true, executes the first
 suite; if the expression is false (which may be the first time it is tested) the
-suite of the :keyword:`!else` clause, if present, is executed and the loop
+:keyword:`!else` clause's suite, if present, is executed and the loop
```

Comment:
The same that the above comment

Reply:
@eamanu Same comment as above.

Reply:
Thanks for the clariftication @maggyero  :)